### PR TITLE
chore: Add error message for SSO config step 3 authorization checkbox

### PR DESCRIPTION
**Pull request by marlonkeating**
[Jira Ticket](https://2u-internal.atlassian.net/browse/ENT-7984)

# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/forms/ValidatedFormCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  };
+
+  return (
+    <>
+      <Form.Label {...formCheckboxProps}>{formCheckboxProps.label}</Form.Label>
+      <Form.Checkbox {...formCheckboxProps} checked={value} onChange={onChange} />
+      {formCheckboxProps.fieldInstructions && (
+        <Form.Text>{formCheckboxProps.fieldInstructions}</Form.Text>
+      )}
+      {showErrors && showError && (
```
Comment (kiram15) [inline src/components/forms/ValidatedFormCheckbox.tsx:53]: Why is there showErrors twice?

Comment (kiram15) [reply src/components/forms/ValidatedFormCheckbox.tsx:53]: Ok reading this again, I see that its two different variables, but I think the {showError} below is confusing cause I would assume that's a boolean and not an error message

**Review by kiram15 — APPROVED** [review]
Other than one nit about variable names it looks good!

File: src/components/forms/ValidatedFormCheckbox.tsx (hunk 1 above)
Comment (marlonkeating) [reply src/components/forms/ValidatedFormCheckbox.tsx]: That's a good point, went ahead and renamed the variable.

**Pull request merged**